topics/food.html

```diff
@@ -0,0 +1,19 @@
+<!doctype html>
+<html lang="en">
+<head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="description" content="">
+    <title>Foods</title>
+</head>
+<body>
+    <h1>Favorite Foods</h1>
+   
+    <ul>
+		<li> Arnold's favorite food is Curry!</li>
```

Comment:
For this, just put in Curry (perhaps specify what type... green)

via github we can determine who added this line... so attribution is stored there